assets/scripts/airport.js

```diff
@@ -1346,6 +1346,7 @@ function airport_init() {
   airport_load('sbgr', "beginner", "Aeroporto Internacional de São Paulo/Guarulhos");
   airport_load('tncm', "easy", "Princess Juliana International Airport");
   airport_load('uudd', "easy", "Moscow Domodedovo Airport");
+  airport_load('vobl', "hard", "Kempegowda International Airport Bengaluru");
```

Comment:
@n8rzz I'm rebasing any remaining pull requests I have anyways.

I'm unaware of what you mean by 'npm commands', all I know about loading an aerodrome is that you have to add it in with the new format in the file `bundle.js`. Could you explain what you mean?